config/routes.rb

```diff
@@ -80,12 +80,7 @@
              }
       end
 
-      resources :guide_parts do
-        member do
-          get "new"
-          post "create"
-        end
-
+      resources :guide_parts, only: %i[new create show edit update show] do
```

Comment:
Typo: `show` appears in the list twice.
```suggestion
      resources :guide_parts, only: %i[new create show edit update] do
```